diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/comments/CommentedCodeTest.kt

```diff
@@ -72,9 +73,7 @@ class CommentedCodeTest : LintTestBase(::CommentsRule) {
            |//    println("This is a test string")
            |    return 0
            |}
-        """.trimMargin(),
-            LintError(4, 1, ruleId, "${COMMENTED_OUT_CODE.warnText()} println(a + 42)", false)
```

Comment:
Why is there no error here? Do you look for `=` in all lines or jsut the first?

Reply:
Why should there be a mistake here? There isn't `=`

Reply:
Right, github has folded two tests and I looked in the wrong place, nvm.